cmd/stack/deploy.go

```diff
@@ -58,7 +56,7 @@ func deploy(ctx context.Context) *cobra.Command {
 			if err != nil {
 				return err
 			}
-			c, config, err := okteto.NewK8sClientProvider().Provide(kubeconfig.Get(config.GetKubeconfigPath()))
```

Comment:
Could you make a quick search to see if there are other places using `okteto.NewK8sClientProvider().Provide(kubeconfig.Get(config.GetKubeconfigPath()))`?